glide.yaml

```diff
@@ -28,3 +28,5 @@ testImport:
 - package: github.com/smartystreets/goconvey
   subpackages:
   - convey
+- package: github.com/smartystreets/assertions
+  version: f44b269b51f6f75a2452c14c4ee76c114ad92528
```

Comment:
this :warning: fixes builds like https://travis-ci.org/github/remerge/rex/builds/750210388 on CI. The reason is that `goconvey` depends on the latest version of `assertions`. The latest version depends on `errors.Is()` which is not available for go 1.11.

I intentionally avoid bump of go version for `rex` since it is deprecated anyway and could bring more issues rather than solutions.

Reply:
@thekondor we should not use goconvey at all - instead of adding new dependency `github.com/smartystreets/assertions`

Also changing rex is waisting time imho - we are not using it since 2018 except couple of places

see https://remerge.slack.com/archives/C029JFQCE/p1608026359418500 

so we should better spend time of remove rex dependendy from there and then remove whole repo... 

WDYT?

Reply:
@gburanov 

1. `github.com/smartystreets/assertions` is **ALREADY** a dependency used by `goconvey`. And the latest version of this dependency is fetched which is incompatible with go `1.11` used in `rex`. Nothing new is added
2. `rex` removal is another kind of task with different efforts to apply. If you take a look at https://github.com/search?q=import+%22github.com%2Fremerge%2Frex%22&type=code you'll see, that amount of work to remove is `rex` is much more

Since this is a cleanup from xmas-party, I'd prefer to go way from this PR.